<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Gaylu" failures="1" tests="2" name="Test(failed)" time="22.0" errors="0" timestamp="2025-03-07T20:01:12 EST">
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verify_account_registration" time="2.206"/>
  <testcase classname="testCases.TC002_LoginTest" name="verify_login" time="11.332">
    <failure type="java.lang.AssertionError" message="Login Passed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Login Passed expected [true] but found [false]
at testCases.TC002_LoginTest.verify_login(TC002_LoginTest.java:55)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_login -->
</testsuite> <!-- Test(failed) -->
